docs/guides/admin/docs/modules/mediamodule.configuration.md

```diff
@@ -11,4 +11,4 @@ The configurations for the media module are done for each tenant. So the configu
     - Default: Opencast logo
 - `prop.player`
     - The player that should be use to play the videos.
-    - Default: Theodul player
+    - Default: Paella player
```

Comment:
Interesting. This configuration does not exist any longer.
The only part which still exists is:

https://github.com/opencast/opencast/blob/d2ab45a9fd345b4bd3fe7bc5160de6446b2d91f2/etc/ui-config/mh_default_org/engage-ui/config.yml#L1-L3

For the player, the media module uses the `/play` endpoint which automatically redirects to the default player.

Reply:
Which loads this config item, doesn't it?  That's what the code looks like at any rate.

Reply:
I'm slightly confused. The [docs you adjusted](https://github.com/opencast/opencast/blob/cb19e3c304404795a04a225feeadfade4aa0cf24/docs/guides/admin/docs/modules/mediamodule.configuration.md) is supposed to describe the media module configuration which does not contain a player configuration and now reside at a different location.

Reply:
I found [https://github.com/opencast/opencast/blob/cb19e3c304404795a04a225feeadfade4aa0cf24/modules/engage-ui/src/main/java/org/opencastproject/engage/ui/PlayerRedirect.java](this) which makes it look to me like it's still in use.  Whether that's true or not I guess I didn't actually check, but it looks like it should be in use...

Reply:
It is. But not in the media mudule :D 
Anyway, let's get it in like this. The other parts are something we can fix later.